libraries/AP_HAL_ChibiOS/hwdef/common/common_extf_h750.ld

```diff
@@ -212,6 +228,7 @@ SECTIONS
         lib/lib*.a:vector3.*(.rodata*)
         lib/lib*.a:matrix3.*(.rodata*)
         *libm.a:*(.rodata*)
+        ArduCopter/Copter.*(.rodata*)
```

Comment:
I think this is for the scheduler table, what about AP_Vehicle?

Reply:
Yes, it's for the scheduler table. It would probably make sense to add all of them and group them together. Does the code below look correct to you?

```
AntennaTracker/Tracker.*(.rodata*)
AP_Vehicle/AP_Vehicle.*(.rodata*)
ArduCopter/Copter.*(.rodata*)
ArduPlane/ArduPlane.*(.rodata*)
ArduSub/ArduSub.*(.rodata*)
Blimp/Blimp.*(.rodata*)
Rover/Rover.*(.rodata*)
```

Reply:
Makes sense to me - with a comment

Reply:
@andyp1per WDYT? Should I add the above change?

Reply:
yes

Reply:
> yes

Done :)